<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>SetTabMidiBase</class>
  <widget class="QWidget" name="SetTabMidiBase" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>352</width>
        <height>163</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>SetTabMidi</string>
    </property>
    <layout class="QGridLayout" >
      <property name="margin" >
        <number>11</number>
      </property>
      <property name="spacing" >
        <number>6</number>
      </property>
      <item row="4" column="0" >
        <widget class="QLabel" name="TextLabel5" >
          <property name="text" >
            <string>Transpose:</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="3" column="0" >
        <widget class="QLabel" name="TextLabel4" >
          <property name="text" >
            <string>Chorus:</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="4" column="2" >
        <widget class="QLCDNumber" name="LCDTranspose" >
          <property name="numDigits" >
            <number>3</number>
          </property>
        </widget>
      </item>
      <item row="0" column="1" >
        <widget class="QSlider" name="SliderVolume" >
          <property name="maximum" >
            <number>127</number>
          </property>
          <property name="pageStep" >
            <number>5</number>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </widget>
      </item>
      <item row="0" column="0" >
        <widget class="QLabel" name="TextLabel1" >
          <property name="text" >
            <string>Volume:</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="4" column="1" >
        <widget class="QSlider" name="SliderTranspose" >
          <property name="minimumSize" >
            <size>
              <width>85</width>
              <height>0</height>
            </size>
          </property>
          <property name="minimum" >
            <number>-12</number>
          </property>
          <property name="maximum" >
            <number>12</number>
          </property>
          <property name="pageStep" >
            <number>1</number>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </widget>
      </item>
      <item row="2" column="2" >
        <widget class="QLCDNumber" name="LCDReverb" >
          <property name="numDigits" >
            <number>3</number>
          </property>
        </widget>
      </item>
      <item row="1" column="0" >
        <widget class="QLabel" name="TextLabel2" >
          <property name="text" >
            <string>Pan:</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="2" column="1" >
        <widget class="QSlider" name="SliderReverb" >
          <property name="maximum" >
            <number>127</number>
          </property>
          <property name="pageStep" >
            <number>5</number>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </widget>
      </item>
      <item row="3" column="1" >
        <widget class="QSlider" name="SliderChorus" >
          <property name="maximum" >
            <number>127</number>
          </property>
          <property name="pageStep" >
            <number>5</number>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </widget>
      </item>
      <item row="2" column="0" >
        <widget class="QLabel" name="TextLabel3" >
          <property name="text" >
            <string>Reverb:</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="3" column="2" >
        <widget class="QLCDNumber" name="LCDChorus" >
          <property name="numDigits" >
            <number>3</number>
          </property>
        </widget>
      </item>
      <item row="0" column="2" >
        <widget class="QLCDNumber" name="LCDVolume" >
          <property name="numDigits" >
            <number>3</number>
          </property>
        </widget>
      </item>
      <item row="1" column="2" >
        <widget class="QLCDNumber" name="LCDPan" >
          <property name="numDigits" >
            <number>3</number>
          </property>
        </widget>
      </item>
      <item row="1" column="1" >
        <widget class="QSlider" name="SliderPan" >
          <property name="minimum" >
            <number>-64</number>
          </property>
          <property name="maximum" >
            <number>64</number>
          </property>
          <property name="pageStep" >
            <number>4</number>
          </property>
          <property name="value" >
            <number>0</number>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <connections>
    <connection>
      <sender>SliderVolume</sender>
      <signal>valueChanged(int)</signal>
      <receiver>LCDVolume</receiver>
      <slot>display(int)</slot>
    </connection>
    <connection>
      <sender>SliderReverb</sender>
      <signal>valueChanged(int)</signal>
      <receiver>LCDReverb</receiver>
      <slot>display(int)</slot>
    </connection>
    <connection>
      <sender>SliderChorus</sender>
      <signal>valueChanged(int)</signal>
      <receiver>LCDChorus</receiver>
      <slot>display(int)</slot>
    </connection>
    <connection>
      <sender>SliderTranspose</sender>
      <signal>valueChanged(int)</signal>
      <receiver>LCDTranspose</receiver>
      <slot>display(int)</slot>
    </connection>
    <connection>
      <sender>SliderPan</sender>
      <signal>valueChanged(int)</signal>
      <receiver>LCDPan</receiver>
      <slot>display(int)</slot>
    </connection>
  </connections>
</ui>
